src/createinstancedialog.h

```diff
@@ -0,0 +1,138 @@
+#ifndef MODORGANIZER_CREATEINSTANCEDIALOG_INCLUDED
+#define MODORGANIZER_CREATEINSTANCEDIALOG_INCLUDED
+
+#include <QDialog>
+
+namespace MOBase { class IPluginGame; }
+namespace Ui { class CreateInstanceDialog; };
+namespace cid { class Page; }
+
+class PluginContainer;
+class Settings;
+
+class CreateInstanceDialog : public QDialog
+{
+  Q_OBJECT
+
+public:
+  enum Types
+  {
+    NoType = 0,
+    Global,
+    Portable
+  };
+
+  struct Paths
+  {
+    QString base;
+    QString downloads;
+    QString mods;
+    QString profiles;
+    QString overwrite;
+    QString ini;
+
+    auto operator<=>(const Paths&) const = default;
```

Comment:
Are we ready to use C++20 features?

Reply:
Spaceship has been available since 2019 afaik, but all I have here is the preview version. If you have RTM, can you do a quick check to make sure? It'll need `/std:c++latest`.

Reply:
It works with 16.7.7.

Reply:
Great, thanks. I'll close this since we have the flag and 2019 supports it.

Reply:
Works fine here